### PR TITLE
Increase cross-compatibility

### PR DESCRIPTION
**Pull request by mitchhentges**
Replace "selector" with "role"
Make accelerator "CommandOrControl" for maximum compatibility

### CODE REVIEW

**Comment by mitchhentges** [comment]
See [menu-item](https://github.com/atom/electron/blob/master/docs/api/menu-item.md)
and [accelator](https://github.com/atom/electron/blob/master/docs/api/accelerator.md)
for what these changes do

**Comment by mitchhentges** [comment]
Oh, also fixes #21

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@ build
 package
 node_modules
 binaries
+*.iml
```
Comment (nerdbeere) [inline .gitignore]: Could you please also remove this here?

Comment (mitchhentges) [inline .gitignore]: Derp, removed.

**Pull request merged**